src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/EventProvider.cs

```diff
@@ -1145,6 +1173,23 @@ internal unsafe bool WriteEvent(ref EventDescriptor eventDescriptor, IntPtr even
             return true;
         }
 
+#if !ES_BUILD_STANDALONE
+        /// <summary>Workaround for inability to stackalloc object[8].</summary>
```

Comment:
Could we add a comment about why this is specifically _eight_ objects (because`EtwAPIMaxRefObjCount == 8`)?

Reply:
Sure.